data/schema/filters/abilities.cfg

```diff
@@ -10,7 +10,7 @@
 	{SIMPLE_KEY replacement_type string_list}
 	{SIMPLE_KEY alternative_type string_list}
 	{SIMPLE_KEY active_on ability_context}
-	{SIMPLE_KEY value s_int_range_list}
+	{SIMPLE_KEY value s_int_range_list_default}
```

Comment:
Now `add`, `sub`, `multiply` and `divide` also support "default".

Reply:
No because none had def value specified

Reply:
Default chexked with "default" if and only if default value is specified like in "value" attrbute for smoe ablities type but not for add sub etc, add don't support "default"

Reply:
added to shema